### PR TITLE
Remove `restartdns`: Redux

### PR DESCRIPTION
**Pull request by PromoFaux**
### **What does this PR aim to accomplish?:**

Missed some references to `restartdns` in https://github.com/pi-hole/pi-hole/pull/5780

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Review by rdwebdesign — APPROVED** [review]

File: pihole [hunk 1]
```diff
@@ -293,7 +293,6 @@ Options:
   Try 'pihole logging --help' for more information."
     exit 1
   fi
-  restartDNS
```
Comment (yubiuser) [inline pihole:296]: FTL will restart itself when required after logging has changed?

Comment (PromoFaux) [reply pihole:296]: I believe so... paging FTL master @DL6ER for confirmation

Comment (rdwebdesign) [reply pihole:296]: Yes.
The `dns.queryLogging` config has the flag `FLAG_RESTART_FTL` set:

https://github.com/pi-hole/FTL/blob/6c0c87547653e3bb33340dfa61c06f7b0cb88d7b/src/config/config.c#L560-L565

Comment (DL6ER) [reply pihole:296]: :) Everything has been said

**Pull request merged**